src/index.ts

```diff
@@ -1,7 +1,11 @@
-import { makeCiCli } from "@useoptic/api-checks/build/ci-cli/make-cli";
-import { newSnykApiCheckService } from "./service";
+import { makeCiCliWithNamedRules } from "@useoptic/api-checks/build/ci-cli/make-cli";
+import { v3Rules } from "./v3-rules";
+// import { internalApiRules } from './internal-api-rules';
 
-const apiCheckService = newSnykApiCheckService();
-const cli = makeCiCli("sweater-comb", apiCheckService);
+const cli = makeCiCliWithNamedRules("sweater-comb", {
+  default: v3Rules(),
+  "v3-apis": v3Rules(),
+  // "internal-apis": internalApiRules(),
+});
 
```

Comment:
@cmars I commented out `internal-apis` rules so nobody tries using it before it's ready